src/networking.c

```diff
@@ -4488,7 +4489,7 @@ int postponeClientRead(client *c) {
  * it can safely perform post-processing and return to normal synchronous
  * work. */
 int handleClientsWithPendingReadsUsingThreads(void) {
-    if (!server.io_threads_active || !server.io_threads_do_reads) return 0;
+    if (server.io_threads_active_num == 1 || !server.io_threads_do_reads) return 0;
```

Comment:
This is not related to your change but the original logic doesn't seem right to me. With this logic the activation of io-threads-do-read implicitly depends on the activation of io-threads-do-write, which should've been two orthogonal decisions IMO. We can address this in a different PR.  
```suggestion
    if (!server.io_threads_do_reads) return 0;
    adjustIOThreadCount();
    if (server.io_threads_active_num == 1) return 0;
```

Reply:
Sure, I will keep the logic as origin, and open another issue to address this.